tests/console/register_original_system.pm

```diff
@@ -0,0 +1,34 @@
+# Copyright 2018 SUSE LLC
```

Comment:
why did you create a new test module? wasn't suitable any existing one? or the one that used for re-registration in  the migration that are already in maintenance?

Reply:
I created a new test module because I added a new variable of SCC_ADDONS_ORIGIN. The already existing test module is using SCC_ADDONS for registration. 
    my $scc_addons = get_var 'SCC_ADDONS_ORIGIN', '';
    register_addons_cmd($scc_addons, $retries) if $scc_addons;